.github/pr_label_checker.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import os
+import re as rgx
+from requests import get, post
+from requests import Response
+from requests.exceptions import HTTPError, RequestException
+import sys
+
+# Constants
+GITHUB_TOKEN=os.environ["GITHUB_TOKEN"]
+GITHUB_API_URL=os.environ["GITHUB_API_URL"]
+GITHUB_SHA=os.environ["GITHUB_SHA"]
+
+BASE_URL=f'{GITHUB_API_URL}/repos/chkim-usgs/ISIS3'
+PULLS_URL=f'{BASE_URL}/pulls'
+COMMITS_URL=f'{BASE_URL}/commits'
+ISSUES_URL=f'{BASE_URL}/issues'
+HEADERS = {
+    "Accept": "application/vnd.github+json",
+    "Authorization" : f"Bearer {GITHUB_TOKEN}",
+    "X-GitHub-Api-Version": "2022-11-28"
+}
+
+def get_prs_associated_with_commit() -> Response:
+    """
+    Get list of PRs associated with commit.
+    """
+    try:
+        response = get(f'{COMMITS_URL}/{GITHUB_SHA}/pulls', headers=HEADERS)
+        response.raise_for_status()
+        return response
+    except HTTPError as he:
+        raise HTTPError("HTTPError in retrieving list of PRs", he) 
+    except RequestException as re:
+        raise RequestException("Unable to retrieve list of PRs associated with commit.", re)
+
+def get_pr_attributes(response: Response) -> tuple:
+    """
+    Get necessary PR attributes.
+    """
+    pull_response_json = response.json()
+    if len(pull_response_json) == 0:
+        # No PRs in commit
```

Comment:
I'm a little confused by what this means.  Generally I view PRs as a list of commits, not the other way around.  Does this mean that the PR has no commit information?